index.js

```diff
@@ -288,11 +290,7 @@ KNXPlatform.prototype.configure = function() {
 			if (that.config.AllowKillHomebridge===true) {
 				response.write(' <br><hr><br><a href="/kill">Kill homebridge</a> by throwing an Error. Use this to restart HomeBridge if you have it configured as a self-starting service ' + ' <BR>');
 			}
-			response.write(`<HR><BR>Available pages are <br>
-					<a href="/list">list devices</a> and <br>
-					<a href="/availservices">list available services</a><br>
-					 <a href="/availcharacteristics">list available characteristics</a>
-					`);
+			response.write('<HR><BR>Available pages are <br><a href="/list">list devices</a> and <br><a href="/availservices">list available services</a><br><a href="/availcharacteristics">list available characteristics</a>');
```

Comment:
Why did you convert all multi-line strings to single line strings? Some issue with your IDE?

Reply:
Ops, i think there was a plugin in my IDE over motivated ...